### DISCUSSION TITLE
x-trap error when using entangle

### DISCUSSION THREAD

**Discussion by bastihilger** [alpinejs/alpine#1982] [7. Bugs (archived)] [answered]
Since a few days (I think after this fix https://github.com/alpinejs/alpine/pull/1965) I get an error message when using x-trap and entangle together. It worked before.

I am using the version 3.2.4 for both alpinejs and x-trap.

This should be enough to reproduce the error ($show is a PHP variable coming from Livewire):

``` html
<div 
    class="
        fixed top-0 w-full h-full overflow-visible
        transition-left duration-300 ease-in-out
        @if($show)
            left-0
        @else
            left-full
        @endif
    "
    x-data="{ open: @entangle('show') }"
>
    <div x-trap="open">
        <p>content removed for brevity</p>
    </div>
</div>
```

This is the error in the console:

```
Uncaught (in promise) TypeError: Cannot read property 'children' of null
    at crawlSiblingsUp (app.js?id=d09d6ad179ad7772b322:552)
    at app.js?id=d09d6ad179ad7772b322:555
    at Array.forEach (<anonymous>)
    at crawlSiblingsUp (app.js?id=d09d6ad179ad7772b322:552)
    at app.js?id=d09d6ad179ad7772b322:555
    at Array.forEach (<anonymous>)
    at crawlSiblingsUp (app.js?id=d09d6ad179ad7772b322:552)
    at app.js?id=d09d6ad179ad7772b322:534
    at runIfTypeOfFunction (app.js?id=d09d6ad179ad7772b322:2231)
    at app.js?id=d09d6ad179ad7772b322:2214

```

Also see this discussion: https://github.com/alpinejs/alpine/discussions/1959

**Comment by MuzafferDede** [comment, marked as answer]
- #1986 should fix this!

**Reply by bastihilger** [reply]
Thank you, that was very quick. I'll check tomorrow if it fixes the problem for me. Cool!

**Reply by bastihilger** [reply]
Wow that was quick. Thanks a lot for looking into this. I hope I will get to check this in the afternoon.

**Reply by MuzafferDede** [reply]
Cool, make sure to give some comment on the PR once you test it so Caleb could reference if PR is solid.

**Comment by bastihilger** [comment]
Checked and commented on PR. Works great again now!
